pywren/scripts/pywrencli.py

```diff
@@ -149,6 +149,26 @@ def create_role(ctx):
     iamclient.RolePolicy(role_name, '{}-more-permissions'.format(role_name)).put(
         PolicyDocument=more_json_policy)
 
+
+@click.command()
+@click.pass_context
+def create_ssh_key(ctx):
+    """
+
+    """
+    config_filename = ctx.obj['config_filename']
+    config = pywren.wrenconfig.load(config_filename)
+
+    region = config['account']['aws_region']
+    ec2 = boto3.client('ec2', region)
+    keyname = config['standalone']['ec2_ssh_key']
+    try:
+        ec2.describe_key_pairs(KeyNames = [keyname])
+    except botocore.exceptions.ClientError as e:
+        if e.response['Error']['Code'] == 'InvalidKeyPair.NotFound':
+            ec2.create_key_pair(KeyName = keyname)
```

Comment:
I see - if we add it as a CLI command then it could be also used by new users who dont know the AWS interface etc. So lets just save it to some (user customizable) location if its not that hard to do it ?